### PR TITLE
EXEC-105: fix documentation

### PR DESCRIPTION
**Pull request by orionlibs**
[EXEC-105](https://issues.apache.org/jira/browse/EXEC-105): fixed documentation

### CODE REVIEW

File: src/site/apt/tutorial.apt [hunk 1]
```diff
@@ -169,7 +169,7 @@ executor.execute(cmdLine, resultHandler);
 
 // some time later the result handler callback was invoked so we
 // can safely request the exit value
-int exitValue = resultHandler.waitFor();
+resultHandler.waitFor();
```
Comment (ecki) [inline src/site/apt/tutorial.apt:172]: I would also show the exitValue = resultHandler.getExitValue(); call as the comment suggests.

Comment (garydgregory) [reply src/site/apt/tutorial.apt:172]: Hi @ecki,
Does the PR look ok to you now?

Comment (ecki) [reply src/site/apt/tutorial.apt:172]: Looks good - maybe in the future we should use snippets or at least have a copy in a test class.

you might need to add the changes.xml entry?

Comment (orionlibs) [reply src/site/apt/tutorial.apt:172]: @ecki updated changes.xml

Comment (garydgregory) [reply src/site/apt/tutorial.apt:172]: In general, don't update changes.xml, if every PR did that, they would all create merge problems.

**Pull request merged**